docs/en/tutorials/getting-started.rst

```diff
@@ -228,33 +227,196 @@ entity definition:
          * @var string
          */
         protected $name;
+    }
 
-        public function getId()
+When creating entity classes, all of the fields should be protected or private (not public).
+
+Adding the Entity behaviors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You have two options to define to define methods in your entity:
+**getters/setters**, or **mutators and DTOs**,
+respectively for **anemic models** or **rich models**.
+
+**Anemic models: Getters and setters**
+
+The most popular method is to create two kinds of methods to
+**read** (getter) and **update** (setter) the object's properties.
```

Comment:
"read" and "update" sounds weird, how about:

- "read" and "write"?
- "share" and "mutate"?

Reply:
I thought read & update were nicer. By adding "write", I'm confused with the fact that we _read something_ but we _write in something_, and I didn't want to bother with my imperfect english 😆 

"share" and "mutate" are closer to the DDD & rich entities approach, and these are words that are possibly unknown by many developers, so I didn't want to add them in the common anemic design